hugr-core/src/types.rs

```diff
@@ -260,15 +260,15 @@ impl TypeEnum {
 ///
 /// Examples:
 /// ```
-/// # use hugr::types::{Type, TypeBound};
-/// # use hugr::type_row;
+/// # use hugr_core::types::{Type, TypeBound};
```

Comment:
examples should not mention `hugr_core`

Reply:
Fixed in 0f23521f212afe234506f9750f3eabf0ef7b4f45